batch7crud-roll4/src/main/webapp/resources/css/layout.css

```diff
@@ -0,0 +1,51 @@
+.wrapper {
+    max-width: 100%;
+}
+
+.header {
+    width: 100%;
+    background: #3c8dbc;
+}
+
+.logo {
+    float: left;
+    width: 230px;
+    line-height: 50px;
+    text-align: center;
+}
+
+.user-info {
+    float: right;
+    width: 16.83748169838946%;
+    text-align: center;
+}
+
+.main {
+    width: 100%;
+}
+
+.col2-left-layout {
+    width: 100%;
+}
+
+.col-left {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Check this potential box model size issue [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Abox-model)